### PR TITLE
Enable rotation of omsagent log with SELinux

### PR DESCRIPTION
**Pull request by sugr4**
@Microsoft/omsagent-devs 
Tested omsagent-logrotate.te for Rhel7/CentOS7/OracleLinux7 
Tested omsagent-logrotate.el6.te for Rhel 6/CentOS6/OracleLinux6
Rhel5/CentOS5 with selinux did not have this issue with logrotate

### CODE REVIEW

File: installer/datafiles/linux_rpm.data [hunk 1]
```diff
@@ -1,10 +1,56 @@
...
+
+%Files
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.fc;                       installer/selinux/omsagent-logrotate.fc;                                 644; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.te;                       installer/selinux/omsagent-logrotate.te;                                 644; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.el6.te;                   installer/selinux/omsagent-logrotate.el6.te;                             644; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.pp;                       intermediate/${{BUILD_CONFIGURATION}}/selinux/omsagent-logrotate.pp;     755; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.el6.pp;                   intermediate/${{BUILD_CONFIGURATION}}/selinux.el6/omsagent-logrotate.pp; 755; root; root
+
+%Directories
+/usr/share/selinux/packages;                                         755; root; root; sysdir
```
Comment (NarineM) [inline installer/datafiles/linux_rpm.data]: Looks good, I have only one question. What happens if /usr/share/selinux does not exist? If someone uninstalls selinux from yum , would that impact omsagent install? Also is this path universal for all rpm distros?

**Comment by NarineM** [comment]
Travis has failed with this PR, can you take a look and update?

File: installer/datafiles/linux_rpm.data (hunk 1 above)
Comment (NarineM) [inline installer/datafiles/linux_rpm.data]: Looks like if /selinux path does not exist, install fails, check it out: 

cp /home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/installer/selinux/omsagent-logrotate.fc /home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/intermediate/Linux_UBUNTU_14.04_x64_64_Debug/selinux.el6
cd /home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/intermediate/Linux_UBUNTU_14.04_x64_64_Debug/selinux.el6; make -f /usr/share/selinux/devel/Makefile
make[1]: Entering directory `/home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/intermediate/Linux_UBUNTU_14.04_x64_64_Debug/selinux.el6'
make[1]: /usr/share/selinux/devel/Makefile: No such file or directory

**Comment by NarineM** [comment]
Can you list all the packages that log rotation is depending on during runtime, if any?

File: installer/datafiles/linux_rpm.data (hunk 1 above)
Comment (sugr4) [inline installer/datafiles/linux_rpm.data]: /usr/share/linux exists on all the rpm based machines that I have, irrespective of whether selinux is enabled by default or not

**Pull request closed without merging**